mir-wizard/src/main/java/org/mycore/mir/wizard/command/MIRWizardGenerateProperties.java

```diff
@@ -65,7 +65,9 @@ public void doExecute() {
                     MCRConfiguration2.set(name, value);
                 }
             }
-            mcrProps.store(new FileOutputStream(file), "Generated by MIR Wizard.");
+            try(FileOutputStream out = new FileOutputStream(file)){
```

Comment:
I think, as a precaution,  `MIRWizardGenerateProperties` should also append to the selected `mycore.properties` file, rather then overwrite it completely. Otherwise, a reordering of `install.xml` could lead to missing properties.

Reply:
done